src/steamcompmgr.cpp

```diff
@@ -4328,7 +4328,7 @@ get_size_hints(xwayland_ctx_t *ctx, steamcompmgr_win_t *w)
 
 	XGetWMNormalHints(ctx->dpy, w->xwayland().id, &hints, &hintsSpecified);
 
-	const bool bHasPositionAndGravityHints = ( hintsSpecified & ( PPosition | PWinGravity ) ) == ( PPosition | PWinGravity );
+	const bool bHasPositionAndGravityHints = hintsSpecified & ( PPosition | PWinGravity );
```

Comment:
I don't think that's equivalent. The previous code checked whether both flags were set, the new code only checks if either flag is set.